portfolio/src/main/java/com/google/sps/servlets/QueryServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps;
+
+import com.google.sps.data.Query;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+@WebServlet("/query")
+public class QueryServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+        loadSamples();
+        response.setContentType("text/plain");
+        response.getWriter().println(Query.makeQuery(samples[0], 7));
```

Comment:
Is the 7 limit for testing? What do we want to do in the general case?

Reply:
I think this is for an old commit. It's addressed in the most recent commit from 3 days ago (I'm using QUERY_SIZE instead of 7 there).